packages/teams-js/test/public/authentication.spec.ts

```diff
@@ -617,6 +637,50 @@ describe('Testing authentication capability', () => {
           });
         });
 
+        it(`authentication.getUser should successfully get user profile with data residency info if dataResidency is provided by hosts`, (done) => {
```

Comment:
Any particular reason to use functional string here, I don't see any variables in the test title